tests/nvapi_d3d11_mocks.cpp

```diff
@@ -175,6 +175,15 @@ class D3D11DxvkDeviceContextMock : public mock_interface<ID3D11DxvkDeviceContext
     IMPLEMENT_MOCK6(MultiDrawIndexedIndirectCount);
     IMPLEMENT_MOCK3(SetDepthBoundsTest);
     IMPLEMENT_MOCK1(SetBarrierControl);
+    IMPLEMENT_MOCK3(GetCudaTextureObject);
```

Comment:
There are expected to be broken right now...

Reply:
Thanks for the heads up, perfectly cool. Let me know when you want me to look at your work again or when I can somehow assist.
(Unfortunately I have to approve CI runs manually every time when you push :( )

Reply:
D'oh!  Well, thanks for doing it!